packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -302,6 +325,18 @@ class OpPerfTelemetry {
 				opPerfData: {},
 			});
 		}
+
+		if (message.type == MessageType.NoOp) {
+			// Count the number of no-ops submitted by this client.
+			// The value is reset when we log the OpStats sampled event.
+			this.noOpCount++;
+		}
+
+		this.opsLogger.sendPerformanceEvent({
```

Comment:
Shouldnn't this be done on op received and not submitted?

Reply:
This is beforeOpSubmit() so probably fine (all of this code might need to be revisited when we get to hide DeltaManager)

Reply:
That said, one keynote - this will count all ops after compression / chunking. Might be Ok, but it will not tell us how many ops that represents from runtime perspective.

Reply:
But aren't no ops sent in response to op processed? What's the relation between ops submitted and no-op?
I thought we want to know how frequently noops are sent as ops are processed.